pkg/check/gsoc/gsoc.go

```diff
@@ -121,7 +121,9 @@ func run(ctx context.Context, uploadClient *bee.Client, listenClient *bee.Client
 	if err != nil {
 		return err
 	}
-	resourceId, socAddress, err := mineResourceId(ctx, addresses.Overlay, privKey, 1)
+	depth := 6
```

Comment:
Why depth of 6?

Reply:
maybe the depth should be configurable, so we can 1 for the CI, and and a different value for the testnet?

Reply:
for depth, maybe commitment depth can be used from the /status endpoint?

Reply:
It can be retrieved from [here](https://github.com/ethersphere/beekeeper/blob/81c573a38b723b1b1b16d176948e3a8c47d93817/pkg/bee/client.go#L1022)